### ISSUE TITLE
Issue when finding element using custom XPATH

### ISSUE THREAD

**Issue by naveen-hitachi** [seleniumbase/SeleniumBase#1917]
In Seleniumbase, when using this XPATH `//*[text() = 'Alarm']` Element is visible
But the same element written like this is not present: `//a[@aria-expanded='false' or not(@aria-expanded)]//*[text() = 'Alarm']`

For some reason Seleniumbase is not able to identify this custom XPATH and I tried similarly to use CSS but result is still the same. 
Element is visible: `span:contains('Alarm')`
Element not visible: `a[aria-expanded='false'], a:not([aria-expanded]) span:contains('Alarm')`

This XPATH works fine when searching with Chrome Inspect and the code has ample wait for element to be visible. I also checked for opacity attribute and found no property for this element. The element is actually visible in the UI but not detected by SeleniumBase. And I have to use the custom xpath to find if the menu option is already expanded or not. 

The html appears like this:
```
<li class="false" style="margin-bottom: 8px;">
  <a aria-expanded="false"><img src="data:image/svg"><span>Alarm</span></a>
</li>
```

**Comment by mdmintz** [comment]
I would stick with ``TAG:contains("TEXT")`` as the CSS Selector to use, as the xpath you're trying to use seems overly complex. Note that some operations require CSS selectors instead of XPath selectors, so if there's no direct way to convert an XPath Selector to a CSS one, then you may experience issues. Use either simple XPath (if you must use XPath), or use CSS Selectors. And note that the ``TAG:contains("TEXT")`` selector format is unique to jQuery and SeleniumBase Selectors, but is not a standard CSS Selector.

**Comment by naveen-hitachi** [comment]
Thanks for the quick response. 

I was able to resolve by reversing the locator like `//*[text() = 'Alarm']/parent::a`
